src/state/ducks/Claims.ts

```diff
@@ -416,7 +416,7 @@ const getCTypeHash = (
 
 const getClaimsByCTypeHash = createSelector(
   [getClaims, getCTypeHash],
-  (entries: Entry[], cTypeHash: ICType['cType']['hash']) =>
+  (entries: Entry[], cTypeHash: ICType['ctypeHash']) =>
```

Comment:
this `ICType` is not the one coming from the sdk. I can't see `ctypeHash` on it

Reply:
Yea, I think this is similar to above with the auto-filling when changing something else.

Reply:
I don't know, if we want to change the `ICType` to the one from the SDK. There is no need for this PR to change it

Reply:
I have reverted the changes.